02_bubble/amitkrishna/bubble_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var sortingTests = []struct {
+	n        []int // input
+	expected []int // expected result
+}{
+	{[]int{3, 2, 1, 5}, []int{1, 2, 3, 5}},
+	{[]int{3, 2, 1, 5, 8}, []int{1, 2, 3, 5, 8}},
+}
+
+func TestBubbleSortOutput(t *testing.T) {
+	r := require.New(t)
+	for _, tt := range sortingTests {
+		actual := bubble(tt.n)
+		if reflect.DeepEqual(actual, tt.expected) {
```

Comment:
Use [ElementsMatchf](https://godoc.org/github.com/stretchr/testify/require#ElementsMatchf) instead.

Reply:
Done

Reply:
Actually, sorry I had this wrong, [require.Equalf](https://godoc.org/github.com/stretchr/testify/require#Equalf) will do.